### ISSUE TITLE
New session per suite

### ISSUE THREAD

**Issue by shuhrat** [gemini-testing/gemini#139]
Now `gemini` uses one session per run, can we configure it to create new session per `suite`?

**Comment by SevInf** [comment]
We can add the option which can enable this, but I don't think we ever can switch to such behavior by default.
Note, that creating new session is much more expensive operation then just opening the url in existing one, so it will make overall test execution time significantly slower. Some parallelization might help to speed things up (AFAIK @hatroman works on something like this), but in case of limited capacity (saucelabs and other cloud WebDriver sevices, for example) this can make tool completely unusable.

In fact, we did this prior to `0.5.0` and `bem-components` tests on sauclebs (which allows only 2 browsers in parallel for open-source projects) run over 50 minutes.

**Comment by SevInf** [comment]
Solution has been merged into master. To switch to 1 browser per suite can be enabled with `sessionMode: perSuite` in config file.

**Comment by shuhrat** [comment]
![impressed](https://raw.githubusercontent.com/jglovier/gifs/gh-pages/impressed/aha.gif)

Thank you!
